src/components/AccountModule/connection-hooks.js

```diff
@@ -63,13 +63,26 @@ export function useWalletConnectionDetails(
 ) {
   const { walletNetworkName, hasNetworkMismatch } = useNetworkConnectionData()
   const theme = useTheme()
-  let connectionDetails = {}
   const isWalletAndClientSynced =
     Math.abs(walletSyncDelay - clientSyncDelay) <= OK_PROVIDER_SYNC_DELAY
   const networkSlowdown =
     walletSyncDelay >= MILD_PROVIDER_SYNC_DELAY &&
     clientSyncDelay >= MILD_PROVIDER_SYNC_DELAY &&
     isWalletAndClientSynced
+  const networkName = network.shortName.toLowerCase()
```

Comment:
Using `network.shortName` instead of `network.type` due to [this issue](https://github.com/aragon/aragon/issues/1472). The reason I'm not using `chainId` is because this is only for local / unknown chains where we don't know its behavior, and the `chainId` *can* be changed with an environment variable.